### DISCUSSION TITLE
Ordering Apps

### DISCUSSION THREAD

**Discussion by Kuuchuu** [Budibase/budibase#4709] [Help] [answered]
Hello! Is there any way at all to set the apps to be in a specific order on the user portal? I can't seem to figure out how it is ordering the apps. It doesn't seem to be alphabetical, but even if it was that wouldn't help my situation. At the moment the Main App is 3rd from last out of 11 apps, which makes it hard for users to figure out where they may want to look first.
Thanks in advance

**Comment by mjashanks** [comment, marked as answer]
Hey @Kuuchuu - you can order and search the apps list at the moment. But I do agree that we could do with more organisation in here. It's definitely something that we will work to improve.

![image](https://user-images.githubusercontent.com/3524181/155964623-bfa617cd-0e87-4e81-a073-6004932cd793.png)
